src/pages/index.tsx

```diff
@@ -68,6 +74,47 @@ const Home = ({ seoMeta }: PageProps) => {
         borderColour={2}
       />
       <CanYouBeAProgrammer />
+      <FlexContainer className='bg-[#923CFF] w-full p-4 flex-col  gap-4'>
+        <FlexContainer className='w-full lg:w-[45%] max-w-[350px] md:max-w-[500px]'>
+          <Image
+            src={COMMUNITY_SECTION.communityBannerImage}
+            fullHeight={true}
+            fullWidth={true}
+            alt='community'
+          />
+        </FlexContainer>
+        <FlexContainer className='max-w-[350px] text-white gap-1  md:max-w-[500px]'>
+          <Text level='h1' className='font-bold text-2xl'>
+            {COMMUNITY_SECTION.communityTitleLine}
+          </Text>
+          <Text level='p' className='text-center'>
+            {COMMUNITY_SECTION.communityDescriptionLine}
+          </Text>
+        </FlexContainer>
+        <FlexContainer className='max-w-[350px] flex-col text-white gap-1  md:max-w-[500px]'>
+          <Text level='h1' className='font-bold text-2xl'>
```

Comment:
![image](https://github.com/The-Boring-Education/TBE-Web/assets/29145481/fd6101f3-f723-49c4-9663-2d7bfff27aae)
Why font-bold text-2xl when you can use strong text as shown in Figma